logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -37,3 +48,48 @@ def format_stackdriver_json(record, message):
     }
 
     return json.dumps(payload)
+
+
+def detect_web_framework():
+    """Detect web framework used in this environment.
```

Comment:
Might as well just return the framework choice in here and drop the module level variables.